server/game/cards/06-CotE/HumbleMagistrate.js

```diff
@@ -0,0 +1,17 @@
+const DrawCard = require('../../drawcard.js');
+const { Players } = require('../../Constants');
+
+class HumbleMagistrate extends DrawCard {
+    setupCardAbilities(ability) {
```

Comment:
Perfectly understandable that you have used `setupCardAbilities(ability)` (a lot of cards still use it and the docs say to use it) but can you please change to use `AbilityDsl`.  Take a look at one of the newer implemented cards to hopefully see what I mean https://github.com/gryffon/ringteki/blob/cote/server/game/cards/06-CotE/DaringChallenger.js

Reply:
I updated the card to use abilityDsl instead. I hope I did this correctly this time.